src/main/java/com/laborete/LaboreteAPI/profile/mappers/MapStructMapper.java

```diff
@@ -0,0 +1,13 @@
+package com.laborete.LaboreteAPI.profile.mappers;
+
+import com.laborete.LaboreteAPI.profile.entity.UserEntity;
+import com.laborete.LaboreteAPI.profile.models.UserDTO;
+import org.mapstruct.Mapper;
+
+@Mapper(
+        componentModel = "spring"
+)
+public interface MapStructMapper {
```

Comment:
Please name this mapper `UserMapper`. Usually we create a separate mapper for each entity.